src/index.html

```diff
@@ -11,7 +11,234 @@
+      method="post"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <div>
+          <label>
+            Surname:
+            <input
+              type="text"
+              name="surname"
+              required
+              autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="form personal">
+          <label>
+            Name:
+            <input
+              type="text"
+              name="name"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+
+        <div class="form personal">
+          <label>
+            How old are You?
+            <input
+              type="number"
+              name="age"
+              value="12"
+              required
+              min="1"
+              max="100"
+            >
+          </label>
+        </div>
+
+        <div class="form personal">
+          <label>
+            Full date of birth:
+            <input
+              type="date"
+              name="date"
+              required
+            >
+          </label>
+        </div>
+
+        <div class="form personal">
+          <label>
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="term"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="form-field">
+        <legend>Registration</legend>
+        <div>
+          <label>
+            E-mail:
+            <input
+              type="email"
+              name="E-mail"
+              placeholder="email@example.com"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+
+        <div>
+          <label>
+            Password:
+            <input
+              class="password"
+              type="password"
+              name="password"
+              required
+              autocomplete="off"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="form-field">
+        <legend>An interesting fact about you!</legend>
+        <div>
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="y"
+            >
+              Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="n"
+            >
+            No
+          </label>
+        </div>
+
+        <div class="form">
+          <label>
+            What is your favorite color?
+            <input
+              type="color"
+              name="color"
+            >
+          </label>
+        </div>
+
+        <div class="form">
+          <label>
+            What time do you go to bed?
+            <input
+              type="time"
+              name="time to bed"
```

Comment:
<img width="1009" alt="Screenshot 2023-03-25 at 17 34 18" src="https://user-images.githubusercontent.com/32853034/227730081-dcd58fbe-d366-4649-833b-e43d3704ead8.png">

```suggestion
              name="timeToBed"
```

fix it everywhere